### DISCUSSION TITLE
Open Mic Session

### DISCUSSION THREAD

**Discussion by kishanrajput23** [scaleracademy/open-source-projects#137] [Ideas]
# Introduction!!
So, it's quite obvious that in our community there are two types of people, the first one is who can freely come over on stage and speak on any topic and the rest of the people has some fear inside even after having all knowledge of particular topic.

## Why?
So, the main idea behind this project is to create a platform where each and everyone get a fair chance to deliver their ideas or thoughts on any particular topic.

Now the question arises that how can we choose a person from the community who's going to be speak in open mic session. So, for that we'll using random selection method where each day one person get selected via bot and he\she will get chance to speak in open mic session.

## Key Points :-
- We'll be rewarding each and every person who complete this task successfully by giving them a particular number of tokens or any prize.
- These sessions help individual's to grow their public speaking skills and interpersonal skills.
- Rest of the community members get chance to learn from others mistake.
- These sessions will be under maintained by Scaler Admins.
- In open mic session the topic would be choose by speaker itself.
- Language : English preferred but not limited to.
- At the end of the session we'll share a google form in which we rate a particular day's speaker on the basis of their topic, delivery, fluency, etc.

**Note** : For those people who will not take this task/challenge seriously then we'll cut a certain number of tokens from their account. The fear of losing tokens,  motivates them to try at least.

That's it from my side. I hope you all love this idea. Please feel free to drop your thoughts below. We'll do healthy discussion over here.

Thank You!!
Kishan

**Comment by shloksuman** [comment]
Nice idea!
